omnigan/trainer.py

```diff
@@ -406,39 +426,45 @@ def log_comet_images(self, mode, domain):
                     target = im_set["data"][update_task].unsqueeze(0).to(self.device)
                     task_saves = []
 
-                    if update_task != "x":
-                        if update_task not in save_images:
-                            save_images[update_task] = []
-                        prediction = self.G.decoders[update_task](self.z)
-                        if update_task == "s":
-                            if domain == "s":
-                                target = (
-                                    decode_segmap_unity_labels(target, domain, True)
-                                    .float()
-                                    .to(self.device)
-                                )
-                            prediction = (
-                                decode_segmap_unity_labels(prediction, domain, False)
+                    if update_task == "x":
```

Comment:
instead of having everything under `if task != "x"` I changed to `if update_task == "x": continue` to have less indentation (and more clarity)